tests/plugins/embedbase/definition.js

```diff
@@ -300,7 +300,7 @@ bender.test( {
 			that.listeners.push( widget.on( 'handleResponse', function( evt ) {
 				eventsCount += 1;
 
-				evt.data.html = evt.data.response.html + '<p>customized</p>';
```

Comment:
This use case is wrong. When widget removed and new one is created from same url, then cached response is used. so this event won't be fired second time. If `evt.data.html` is modified then the second widget will be different from the first one. Only way to make it work as suggested in comment below is to modify the response itself, so `_getCachedResponse` will return modified html.